From e9bc8f6deda0e76d6f02f9e19e19ef95c1aa07e1 Mon Sep 17 00:00:00 2001
From: Rei Oliveira <Reinildes.JoseMateusOliveira@windriver.com>
Date: Wed, 19 Oct 2022 12:54:01 -0300
Subject: [PATCH] Increase timeout from 40s to 100s

This puppet module is used by the bootstrap manifest to execute
openstack commands. After the change to debian, this 40s timeout
is not enough for some types of hardware.

Signed-off-by: Rei Oliveira <Reinildes.JoseMateusOliveira@windriver.com>
---
 lib/puppet/provider/openstack.rb | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/puppet/provider/openstack.rb b/lib/puppet/provider/openstack.rb
index 3d80d68..fe63b79 100644
--- a/lib/puppet/provider/openstack.rb
+++ b/lib/puppet/provider/openstack.rb
@@ -14,7 +14,7 @@ class Puppet::Provider::Openstack < Puppet::Provider
   commands :openstack_command => 'openstack'
 
   @@no_retry_actions = %w(create remove delete)
-  @@command_timeout  = 40
+  @@command_timeout  = 100
   # Fails on the 5th retry for a max of 212s (~3.5min) before total
   # failure.
   @@request_timeout  = 170
-- 
2.17.1

